frontend/public/style.scss

```diff
@@ -6,7 +6,7 @@
 // Patternfly variables must be imported before Bootstrap
 @import '~patternfly/dist/sass/patternfly/variables';
```

Comment:
I assume you're working on moving the vars for PatternFly 3 and Bootstrap over?

Reply:
yes